app/build.gradle

```diff
@@ -44,13 +44,14 @@ if (project.file('../PiwigoSigning.properties').exists()) {
 }
 
 android {
-    compileSdkVersion 28
+    compileSdkVersion 29
     defaultConfig {
         applicationId "org.piwigo.android"
-        minSdkVersion 14
-        targetSdkVersion 28
+        minSdkVersion 16
```

Comment:
why do we need sdk 16 instead of 14? - Don't get me wrong, I think it is sufficient to support down to 16, so don't waste effort by this. Still I am curios to know why we need it - just to learn something.

Reply:
After some testing it turned out that some features were broken on SDK 14, like navigation.

Also SDK 16 is the requirement as minSdk for some dependencies.

Reply:
ah ok, great. As we are about it: the navigation is something we should care about. Because today we have only the back button to navigate up. But I think we should have a "Up Navigation" in the toolbar.